resources/views/productsingle.blade.php

```diff
@@ -0,0 +1,261 @@
+										<div class="product-des">
+											<!-- Description -->
+											<div class="short">
+												<h4>{{ $product['name'] }}</h4>
+												<div class="rating-main">
+													<ul class="rating">
+														<li><i class="fa fa-star"></i></li>
+														<li><i class="fa fa-star"></i></li>
+														<li><i class="fa fa-star"></i></li>
+														<li><i class="fa fa-star-half-o"></i></li>
+														<li class="dark"><i class="fa fa-star-o"></i></li>
+													</ul>
+													<a href="#" class="total-review">(102) Review</a>
+												</div>
+												<p class="price"><span class="discount">{{ $product['price'] }}€</span><s>{{ $product['sale'] }}</s> </p>
+												<p class="description">{{ $product['description'] }}</p>
+											</div>
+											<!--/ End Description -->
+											<!-- Color -->
+											<div class="color">
+												<h4>Available Options <span>Color</span></h4>
+												<ul>
+													<li><a href="#" class="one"><i class="ti-check"></i></a></li>
+													<li><a href="#" class="two"><i class="ti-check"></i></a></li>
+													<li><a href="#" class="three"><i class="ti-check"></i></a></li>
+													<li><a href="#" class="four"><i class="ti-check"></i></a></li>
+												</ul>
+											</div>
+											<!--/ End Color -->
+											<!-- Size -->
+											<div class="size">
+												<h4>Size</h4>
+												<ul>
+													<li><a href="#" class="one">S</a></li>
+													<li><a href="#" class="two">M</a></li>
+													<li><a href="#" class="three">L</a></li>
+													<li><a href="#" class="four">XL</a></li>
+													<li><a href="#" class="four">XXL</a></li>
+												</ul>
+											</div>
+											<!--/ End Size -->
+											<!-- Product Buy -->
+											<div class="product-buy">
+												<div class="quantity">
+													<h6>Quantity :</h6>
+													<!-- Input Order -->
+													<div class="input-group">
+														<div class="button minus">
+															<button type="button" class="btn btn-primary btn-number" disabled="disabled" data-type="minus" data-field="quant[1]">
+																<i class="ti-minus"></i>
+															</button>
+														</div>
+														<input type="text" name="quant[1]" class="input-number"  data-min="1" data-max="1000" value="1">
+														<div class="button plus">
+															<button type="button" class="btn btn-primary btn-number" data-type="plus" data-field="quant[1]">
+																<i class="ti-plus"></i>
+															</button>
+														</div>
+													</div>
+													<!--/ End Input Order -->
+												</div>
+												<div class="add-to-cart">
+													<a href="#" class="btn">Add to cart</a>
+													<a href="#" class="btn min"><i class="ti-heart"></i></a>
+													<a href="#" class="btn min"><i class="fa fa-compress"></i></a>
+												</div>
```

Comment:
```suggestion
												<div class="add-to-cart">
													<a href="#" class="btn">Add to cart</a>
													<a href="#" class="btn min"><i class="ti-heart"></i></a>
													
												</div>
```
Ogni <a> dev'essere racchiuso dentro un form, con metodo post e action la route che dobbiamo creare. Bisogna inviare l'id del prodotto che si vuole aggiungere alla lista dei preferiti o nel carrello